Commands/SitesMassWordPressUpdateCommand.php

```diff
@@ -471,4 +534,109 @@ private function send_to_slack($text) {
   private function isReport() {
     return isset($this->assoc_args['report']) ? true : false;
   }
+
+  private function isConfigFile() {
+    return isset($this->assoc_args['config-file']) ? true : false;
+  }
+
+  private function getSetting($name, $default_value = false) {
+    return isset($this->assoc_args[$name]) ? true : $default_value;
+  }
+
+  private function isExcluded($site) {
+    if ($this->isConfigFile()) {
+
```

Comment:
please, never leave an if empty, change the order or think about why you are doing that.